.env.dist

```diff
@@ -0,0 +1,2 @@
+export YAHOO_OAUTH_KEY=''
+export YAHOO_OAUTH_SECRET=''
```

Comment:
`.env.dist` = `.env.dist.sample`, and you rename it to .env, which is gitignored :)